lms/djangoapps/mobile_api/models.py

```diff
@@ -2,8 +2,9 @@
 ConfigurationModel for the mobile_api djangoapp.
 """
 from django.db import models
-from mobile_api import utils
+
 from config_models.models import ConfigurationModel
+from mobile_api import utils
```

Comment:
Since we're already in `mobile_api`, I *think* we ought to be able to just have `import utils` for this line.

@jcdyer, you're my go-to for python import standards, can you confirm the best practice?